concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json

```diff
@@ -0,0 +1,98 @@
+{
+    "@context": [
+        "http://www.w3.org/ns/csvw",
+        {
+            "@language": "en"
+        }
+    ],
+    "dialect": {
+        "encoding": "utf-8-sig",
+        "delimiter": "\t",
+        "skipBlankRows": true
+    },
+    "tables": [
+        {
+            "tableSchema": {
+                "columns": [
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[a-zA-Z]+\\-[0-9]{4}\\-[0-9]+[a-z]?\\-[0-9]+[a-z]?$"
+                        },
+                        "name": "ID"
+                    },
+                    {
+                        "datatype": {
+                            "base": "string",
+                            "format": "[0-9\\.]+([a-z\\\u2013]+)?$"
+                        },
+                        "name": "NUMBER"
+                    },
+                    {
+                        "datatype": {
+                            "base": "integer",
+                            "minimum": 1
+                        },
+                        "name": "CONCEPTICON_ID"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "CONCEPTICON_GLOSS"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SEMANTIC_CLASS"
+                    },
+                    {
+                        "name": "TYPICAL_ATTRIBUTE",
+                        "separator": ",",
```

Comment:
Separator is `", "` so it is `,` = comma + ` ` space. 

BTW, try to load the concepticon and you can see what this means for loading the list:

```python
from pyconcepticon import Concepticon
con = Concepticon()
for c in con.conceptlists['Baroni-2011-200'].concepts.values():
    print(c.english, len(c.attributes['cohyponymy']), c.attributes['cohyponymy'][0])
```

If separator works fine, you should see a full word, plus the length of the items in the list.

Reply:
I added the comma + ` ` space in the metadata.json, but the script only returns the first item of each column.

Reply:
First *item* means what? First letter, or first word?

Reply:
First word. For the column TYPICAL_ATTRIBUTE it looks like this
```
alligator 16 aggressive
frog 14 amphibious
lizard 10 brown
snake 24 aggressive
turtle 9 amphibious
dishwasher 14 clean
freezer 17 big
fridge 17 big
oven 12 clean
phone 19 big
radio 16 big
stereo 17 big
stove 12 clean
television 21 addictive
toaster 10 clean
washer 16 big
crow 7 black
dove 5 domestic
eagle 10 aggressive
falcon 8 aggressive
goose 6 aquatic
hawk 10 aggressive
owl 9 brown
penguin 10 Antarctic
pheasant 8 brown
pigeon 7 dirty
robin 7 common
sparrow 6 brown
swan 10 aquatic
```